packages/react-link/SPEC.md

```diff
@@ -0,0 +1,225 @@
+# Link
+
+**GitHub Epic issue** - [Link Convergence #16572](https://github.com/microsoft/fluentui/issues/16572)
+
+## Background
+
+The `Link` component references data that a user can follow by clicking or tapping it.
+
+## Prior Art
+
+### Open UI
+
+The Open UI [Link Research](https://open-ui.org/components/link.research) page (currently in PR: https://github.com/WICG/open-ui/pull/253), shows how the `Link` component is used in UI platforms across the web. The consensus across libraries seems to center around a simple interface with few variants that mostly attempt to support the functionality of the `<a>` HTML tag.
+
+### Comparison of v8 and v0
+
+The existing components are:
+
+- v8 - [Link](https://developer.microsoft.com/en-us/fluentui#/controls/web/link)
+- v0 - v0 does not currently export a `Link` component
+
+## Sample Code
+
+Basic examples:
+
+```jsx
+<Link>This is a link</Link>
+<Link href="https://www.bing.com">This is a link</Link>
+<Link href="https://www.bing.com" disabled>This is a link</Link>
+<Link href="https://www.bing.com" target="_blank">This is a link</Link>
+```
+
+## Variants
+
+### Default vs inline
+
+A `Link` has two decoration variants depending on where it's used.
+
+By `default`, the `Link` should be limited to homogeneous surface areas where everything is clickable in the space and should appear as lacking an underline.
+
+A second `inline` variant is provided for scenarios where body text is going to be used alongside a `Link`. This variant adds an underline to add a separate visual distinction in addition to color for contrast purposes.
+
+### Anchor vs button
+
+A `Link` renders as different HTML tags depending on the whether a value has been passed for the `href` property or not. If a value has been passed to the `href` property, then the `Link` renders as an `<a>` HTML tag. Conversely, if the `href` property is left `undefined` the `Link` renders as a `<button>` HTML tag.
+
+The `Link` can also be custom rendered as something entirely different by replacing the `root` slot with the preferred element to be rendered.
```

Comment:
Should this say "`as` prop" instead of "`root` slot"?

Reply:
Yeah, will change that to say this:

```suggestion
The `Link` can also be custom rendered as something entirely different by replacing the `root` slot with the preferred element to be rendered via the `as` prop.
```